src/schedule-and-details/license-section/index.jsx

```diff
@@ -54,7 +54,7 @@ LicenseSection.defaultProps = {
 LicenseSection.propTypes = {
   license: PropTypes.string,
   onChange: PropTypes.func.isRequired,
-  isEditable: PropTypes.bool,
+  isEditable: PropTypes.bool.isRequired,
```

Comment:
You can remove `isEditable` from the `defaultProps` section if it is now required.